src/Avalonia.Native/AvaloniaNativeApplicationPlatform.cs

```diff
@@ -13,12 +13,21 @@ internal class AvaloniaNativeApplicationPlatform : NativeCallbackBase, IAvnAppli
         void IAvnApplicationEvents.FilesOpened(IAvnStringArray urls)
         {
```

Comment:
This method is never really executed anymore?

Reply:
still executed to maintain legacy behaviour, but it wont call the new uri api codepath from here.